### DISCUSSION TITLE
Use `fileName` instead of `filePath` (a throwback . . .) only when the file directory is the CWD already

### DISCUSSION THREAD

**Discussion by diogo-rossi** [haiiliin/abqpy#3788] [Ideas]
## Summary
This discussion proposes to use only the script's file path "basename" (i.e. the file name), instead of the full absolute file path when running Abaqus command, **only when the script's directory is already the current work directory (CWD)**.

## Introduction
This discussion proposes a small "throwback" to an old execution way, already discussed in #1095, #1103, #1104.
But, the "throwback" is proposed only in a very specific situation: when the script's directory is the already the CWD.

## Motivation
The purpose of this modification is to execute Abaqus in a server through a remote connection (e.g. ssh client) which may have differences in the mounted folders.

## Description of a possible problem
A folder mounted in Windows system may not have the same directory path as the same folder mounted in a Linux server, and the opposite is also true. Then, the execution of the command line 
```cmd
abaqusBATpath cae noGUI=path/to/file/in/windows/script.py
```
may lead to an "File not found" error when the `abaqusBATpath` is calling abaqus to run remotely through a ssh connection (and the python interpreter is on Windows).

## Possible solution
One possible solution to the above problem is to set the abaqus command to use the script's file name without the full absolute file path, **in the situation when the script is in the CWD**. When the script is outside the CWD the error will still happen in a ssh connection, but no error will occur in an normal local abaqus execution. This modification also leads to a simpler command line output when the script is in the CWD.

## Implementation
This simple modification is done on the following snnipet 

https://github.com/haiiliin/abqpy/blob/90bf3ab3dc606ddce3d9f9e47ee7f8cf51fb1db6/src/abqpy/abaqus.py#L37-L39

It is on the the branch

https://github.com/haiiliin/abqpy/tree/_Test-local-file-path

If it is ok, I can open an PR for this.

**Comment by haiiliin** [comment]
How about using `os.path.relpath` always?

**Reply by diogo-rossi** [reply]
Excelente suggestion!

Done on a new branch 

https://github.com/haiiliin/abqpy/tree/_use-relpath-on-execution
